bin/ka-clone

```diff
@@ -24,9 +24,24 @@ def _expanded_home_path(path):
 
 
 def _cli_parser():
+
+
     parser = argparse.ArgumentParser(
         description='Clones and configures a KA repo.',
         usage=_cli_usage())
+    # NOTE(Lilli): Please make sure that the version is always an integer,
+    # and that it stays in sync with the version in the OLC file
+    # https://github.com/Khan/our-lovely-cli/blob/main/cmd/repo-rangers/ranger-version.ts
```

Comment:
This url will work, once I land the corresponding PR in OLC.

Also, how do I disable the "line too long" lint check on this?